### ISSUE TITLE
App closes, after importing database from GitHub play version to Google Playstore version.

### ISSUE THREAD

**Issue by Thunderhawk2001** [XilinJia/Podcini#122]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/XilinJia/podcini/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/XilinJia/Podcini/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

6.14.7

### Where did you get the app from

Google Play

### Problem you may be having, or feature you want

I exported my database from the GitHub version (6.14.8), uninstalled the App, installed the play store testing version, then wanted to import the database. App asks for a restart, I click OK, the App closes but doesn't open again. Trying to start the App again, just shows the splash screen for half a second then closes the app.

Could be related to my prior version being higher, than the play store version?

**Comment by Thunderhawk2001** [comment]
Okay just tried to import the 6.14.8 version database  of the GitHub play app to the 6.14.7 version of the GitHub play app.
Works perfectly. So maybe not the culprit?

**Comment by XilinJia** [comment]
> Okay just tried to import the 6.14.8 version database of the GitHub play app to the 6.14.7 version of the GitHub play app. Works perfectly. So maybe not the culprit?

But you can't import the DB to the play app on Google Play?

**Comment by XilinJia** [comment]
I think the DB of the play app on Google Play is older, lacking commentTime added, so it has harder time importing.  Will update shortly.
Sorry for the trouble.

**Comment by Thunderhawk2001** [comment]
> > Okay just tried to import the 6.14.8 version database of the GitHub play app to the 6.14.7 version of the GitHub play app. Works perfectly. So maybe not the culprit?
> 
> But you can't import the DB to the play app on Google Play?

Yes, exactly. If I stay on the GitHub versions it's working perfectly. Just when I try to switch over to the Google Play store app, I get this issue.

**Comment by XilinJia** [comment]
Updated Google play to 6.15.1, you may see it after a bit while Google reviews it.

BTW, you shouldn't need to remove the existing app (though you'd better back up the DB for safety), you should be able to install the play app on top of the existing app.  Let me know it doesn't work for you.

**Comment by Thunderhawk2001** [comment]
Thank you 😃

Will have a look again with the new version, but I only could uninstall the app in the Google Play store and only after uninstalling did the install option appear.

**Comment by Thunderhawk2001** [comment]
Ok. This time it presented the update button, but failed with "Could not be installed" error message. Does it help you, if I use the "Give Feedback" option, or is it just for Google?

But after uninstalling the previous version and installing the play store version, the import or the database and preferences is working.

**Comment by XilinJia** [comment]
I see.  So the play app "Could not be installed" on top of existing app must be due to different signatures (Play store uses its own signature, not sure how to have it use my own).

Not sure how "Give Feedback" works.  Will try it myself.
